src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -175,7 +175,10 @@ trait BackendCall {
       else
         Future.successful(Option(dockerImage))
 
-    call.task.runtimeAttributes.docker map hashDockerImage getOrElse Future.successful(None) map hashGivenDockerHash
+    if (workflowDescriptor.configCallCaching)
```

Comment:
On the other hand all the `if (workflowDescriptor.configCallCaching)` in the callers isn't an awesome alternative :frowning: